docs/articles/beyond_basics/interactions.md

```diff
@@ -0,0 +1,41 @@
+# Interactions
+
+[Interactions](https://discord.com/developers/docs/interactions/receiving-and-responding#interactions) represent a user interacting with your bot.
+This can happen in different ways. The most prominent case is Application Commands ("slash commands") but also button presses or context menus.
+
+## Recieving interactions
+Discord offers two ways to recieve interactions: Gateway and HTTP webhooks.
+Currently DSharpPlus only supports the first one but there are plans to also integrate webhooks.
```

Comment:
Is this not planned to be merged after #1999? In which case "there are plans to integrate webhooks" would be incorrect.

Reply:
Currently i plan to merge it first and then add updating this in #1999